src/Features/AssignableAsset.php

```diff
@@ -53,9 +53,11 @@ public function canViewItem()
 
         $is_assigned = $this->fields['users_id_tech'] === $_SESSION['glpiID'] ||
             in_array((int) ($this->fields['groups_id_tech'] ?? 0), $_SESSION['glpigroups'] ?? [], true);
+        $is_owned = isset($this->fields['users_id']) && $this->fields['users_id'] === $_SESSION['glpiID'];
```

Comment:
`$is_owned` should check the groups too.

```suggestion
        $is_owned = isset($this->fields['users_id']) && $this->fields['users_id'] === $_SESSION['glpiID'] ||
            in_array((int) ($this->fields['groups_id'] ?? 0), $_SESSION['glpigroups'] ?? [], true);
```

Reply:
This confused me too, but I'm pretty sure the Group field is to add assets to groups rather than meaning a group of users owns it. The spec also only mentions the User field.

The way I understand it from the code, user documentation and the spec for this change:
User in charge - Technician who is in charge of maintaining the asset
Group in charge - Technician group(s) in charge of maintaining the asset
User - The user that currently has the asset assigned to them
Group - The asset group(s) the asset beings to

https://glpi-user-documentation.readthedocs.io/fr/latest/modules/administration/groups.html

I've seen groups that allow both items and users in them being used in the Group field to denote a group of people that are assigned the asset for some shared assets like printers (ex everyone in the HR office uses the HR printer). However, that wouldn't be the intended purpose.

Reply:
group and user (with _tech suffix) means the assets have been assigned (for usage) to you or one of your group
We summarize this by "own" keyword but the behavior is the same for both direct usage and by the group, concerned assets should be owned by the user. Later we may have a page dedicated to list assets owned by the user.

Reply:
> group and user (with _tech suffix) means the assets have been assigned (for usage) to you or one of your group
That isn't the intent I infer from the label. "Technician in charge" and "Group in charge" are very different than the User/Group assigned to the asset to use. In fact, the users_id_tech field specifically restricts the options to users who can own a ticket further indicating that it is for technicians that are responsible for maintenance only. This behavior goes back to at least 0.85.

For example, my office printer may be *used* by me, but the person/group responsible for maintenance/repairs is the printer technician.

The spec for two new permissions (this PR and the previous) seem to have the labels for the new permissions switched (my fault for not saying something earlier I guess). The permissions for the *_tech fields should be Read/Update Owned while this PR should be Read/Update Assigned or Read/Update Used.

> Later we may have a page dedicated to list assets owned by the user.

Not sure I understand because the User form has tabs for both managed assets (when the user is assigned to the _tech fields directly or through a group) and used assets (users_id). The used assets tab actually includes assets assigned through the groups_id field, but that doesn't seem like a correct behavior since the Group field only shows groups with `is_itemgroup` set to yes (So it is an asset group and not necessarily a group that can contain users), while groups_id_tech is limited to groups that can be assigned to a ticket.